articles/tech-resources/change-log-sparkpost-enterprise.md

```diff
@@ -6,6 +6,37 @@ notification: "This Change Log is for SparkPost Enterprise Only"
 
 A running log of new features and capabilities that have been rolled out on the SparkPost Enterprise service. The API reference can be found [here](https://developers.sparkpost.com/api).
 
+
+### December, 2017
+* **Change:** Shorter SparkPost engagement tracking URLs
+  * SparkPost changed our tracking URLs to be shorter. The precise amount of change depends on how much personalization and meta data a customer is sending with their transmission.
+
+
+### November, 2017
+* **NEW:** Additional Open Pixel at message top
+  * SparkPost has added an additional open pixel to the top of each message where open tracking is selected. This is in addition to the open pixel that is normally inserted at the bottom of the message
+  * Customers should expect a new event in the Message Events API and in your webhook events: initial_open
+  * Customer should also expect an additional Metric in the Metrics API: initial_rendered
+  * Rollout of this feature is expected over the first 2 weeks of November
+  
+* **Enhancement:** Self-service for HTTPS tracking links
+  * SparkPost Customers can now setup secure tracking links/domains on their ownusing a third party Content Delivery Network (CDN). We will continue to support Enterprise tenants that require we host their certificates rather than use a CDN.
+  * This is currently only available via API. We will update the WebUI in the next couple of weeks
```

Comment:
Do we always say "WebUI" because there is only one UI....we should probably have a standard and stick with it. Using both could lead to confusion.